ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+- name: 🙋🏾🙋🏼‍Question
+  url: http://discord.eddiehub.org/
```

Comment:
```suggestion
  url: http://discord.eddiehub.org
```

Reply:
Okay, I will make the change.

Reply:
@krishguptadev Then I guess it is an issue in Link Free also. Should I make PR there also?